generator-prisma-client/nextjs-starter-webpack-turborepo/README.md

```
@@ -104,13 +121,15 @@
 
 Note that `__YOUR_PRISMA_POSTGRES_DIRECT_CONNECTION_STRING__` is a placeholder value that you need to replace with the values of your Prisma Postgres direct TCP connection string. The direct connection string has the following structure: `postgres://<username>:<password>@<host>:<port>/<database>`.
 
-Expose the environment variables to your project by running the following command:
+### 3. Build the Prisma Client and the Next.js app
+
+Run:
 
-```bash
-source .envrc
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)